### PR TITLE
UT-Refactor4.Refactor some ut&Add README

### PR DESCRIPTION
**Pull request by Seaven**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/test/java/com/starrocks/sql/plan/DistributedEnvPlanWithCostTest.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ public void testCountDistinctWithGroupHighCountLowCHAR() throws Exception {
     public void testCountDistinctWithGroupHighCountHigh() throws Exception {
         String sql = "select count(distinct P_NAME) from part group by P_PARTKEY;";
         String planFragment = getFragmentPlan(sql);
+        System.out.println(planFragment);
```
Comment (Youngwb) [inline fe/fe-core/src/test/java/com/starrocks/sql/plan/DistributedEnvPlanWithCostTest.java:84]: remove

**Review by stdpain — DISMISSED** [review]

File: fe/fe-core/src/test/java/com/starrocks/qe/ConnectSchedulerTest.java [hunk 2]
```diff
@@ -68,26 +64,6 @@ public void setUp() throws Exception {
 
     @Test
     public void testSubmit(@Mocked ConnectProcessor processor) throws Exception {
-        // mock new processor
```
Comment (Youngwb) [inline fe/fe-core/src/test/java/com/starrocks/qe/ConnectSchedulerTest.java:71]: Why these are useless?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: fe/fe-core/src/test/java/com/starrocks/qe/ConnectSchedulerTest.java (hunk 2 above)
Comment (Seaven) [reply fe/fe-core/src/test/java/com/starrocks/qe/ConnectSchedulerTest.java:71]: To be honest, I don't know, but I'm sure these cases haven't checked the results of processor work

**Review by kangkaisen — APPROVED** [review]

**Review by Youngwb — APPROVED** [review]

**Pull request merged**